src/content/docs/en/pages/devtools/cli/azion-cli-alpha/domains/domains.mdx

```diff
@@ -1,12 +1,16 @@
 ---
-title: Domains
+title: Azion CLI domain
 description: Create, list, describe, update, and delete your domains on the Azion platform through Azion CLI.
 meta_tags: cli, domains, devops
 namespace: documentation_cli_domains
 menu_namespace: cliMenuAlpha
 permalink: /documentation/products/cli/domains-crud/
 ---
 
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case any required field isn't informed through the specific flags, an interactive message prompts and asks for the missing information.
```